client/my-sites/sidebar/sidebar.jsx

```diff
@@ -361,17 +372,20 @@ module.exports = React.createClass( {
 
 		return (
 			<li className={ this.itemLinkClass( [ '/plans' ], linkClass ) }>
-				<a onClick={ this.trackUpgradeClick } href={ planLink }>
+				<a onClick={ this.trackUpgradeClick.bind( this, planLink ) } href={ planLink }>
```

Comment:
don't we try to avoid binding in the render method for performance reasons (real question)? /cc @mtias